46-sugar-ant/technical_specification.md

```diff
@@ -251,6 +254,12 @@ The claims repository also has endpoints at
 
 The routes of these endpoints do not need to be changed. However, these endpoints must now also check that the corresponding claims are bound to IVAs that have been verified for the given user.
 
+Currently, the data stewards are defined via the configuration of the claims repository, which is used to seed the claims repository with data steward claims. This configuration needs to be extended to also include the type and value of an associated IVA. The code that seeds the repository will create not only the data steward claims, but also the associated users and IVAs if these do not exist. However, it will not set the IVA state to `verified`.
+
+This means that at least for one data steward, the verification of the corresponding IVA needs to be set manually in the database. This data steward can then also verify the IVAs of other data stewards via the API. The inter-service integration tests need to set the state of the first data steward via the "state management service" since this cannot be done with pure black-box testing via the API.
+
+Note that currently, this seeding also removes all existing data steward claims. If we decide to keep the data steward claims, we should still not set the IVA to `verified` state on creation, since data stewards who re-created their 2FA token and thereby unverified their IVA could simply remove their IVA and would be re-verified on the next start of the claims repository.
```

Comment:
Not sure why this is stressed here. Why would we ever bootstrap with "verified" state?

Reply:
It would simplify the bootstrapping with the first data steward and as an alternative to SMS for testing. But yes, maybe it's better to remove that sentence completely.

Reply:
changed in 864061